doc/source/changelog.rst

```diff
@@ -16,7 +16,13 @@ Bug Fixes
 - Fix the final time calculation in ``GateCompiler.compile``. (`#162 <https://github.com/qutip/qutip-qip/pull/162>`_)
 - Fix bug for circuit with selected measurement result. (`#167 <https://github.com/qutip/qutip-qip/pull/167>`_)
 - Fix bugs in the analytical pulse shapes. (`#169 <https://github.com/qutip/qutip-qip/pull/169>`_)
-- Fix the swap gate plotting. (`#170 <https://github.com/qutip/qutip-qip/pull/170>`_)
+- Fix the swap gate plotting. (`#170 <https://github.com/qutip/qutip-qip/pull/170>`_) and (`#177 <https://github.com/qutip/qutip-qip/pull/177>`_)
+- Filter the measurement probability (`#182 <https://github.com/qutip/qutip-qip/pull/182>`_)
+- Add a default max_step based on the circuit time (`#183 <https://github.com/qutip/qutip-qip/pull/183>`_)
+
+Miscellaneous
+-------------
+- Fix parsing of package versions in packaging 22. (`#181 <https://github.com/qutip/qutip-qip/pull/181>`_)
```

Comment:
I simplified the fix for this in qutip to just do:
```
version = packaging.version.Version(version_string)
```
which does the right thing too.

Reply:
Okay, so we just missed that `Version` class. I'll include it in another PR. This should work here.